src/components/starter/home_illustration.styles.ts

```diff
@@ -0,0 +1,87 @@
+import { css } from '@emotion/react';
+
+export const homeIllustration = euiTheme => ({
+  homeIllustration: css`
+    position: relative;
+    display: flex;
+    justify-content: center;
+
+    @media (min-width: ${euiTheme.breakpoint.m}px) {
+      justify-content: flex-end;
+    }
+  `,
+  homeIllustrationEffect: css`
+    display: block;
+    position: relative;
+
+    .homeIllustration__EffectSVG {
```

Comment:
Using classes here because of performance issues. If I was using this directly in emotion CSS prop the animation was performing slowly.